platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -112,8 +111,8 @@ public record PlatformSchedulersConfig(
                 TaskSchedulerConfiguration runningEventHasher,
         @ConfigProperty(defaultValue = "SEQUENTIAL CAPACITY(500) UNHANDLED_TASK_METRIC")
                 TaskSchedulerConfiguration issDetector,
-        @ConfigProperty(defaultValue = "SEQUENTIAL_THREAD") TaskSchedulerType hashLoggerSchedulerType,
-        @ConfigProperty(defaultValue = "100") int hashLoggerUnhandledTaskCapacity,
+        @ConfigProperty(defaultValue = "SEQUENTIAL CAPACITY(100) UNHANDLED_TASK_METRIC")
```

Comment:
Was the change from `SEQUENTIAL_THREAD` to `SEQUENTIAL` intentional?

Reply:
Intentional. This thread is just logging, there is no need to have a dedicated thread. Although technically it touches the state, it only touches in memory parts (so we don't have to worry about disk access).